src/VisualStudio/Setup/source.extension.vsixmanifest

```diff
@@ -72,5 +72,6 @@
     <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[17.0,18.0)" DisplayName="Visual Studio core editor" />
     <Prerequisite Id="Microsoft.DiaSymReader.Native" Version="[17.0,18.0)" DisplayName="Windows PDB reader/writer" />
     <Prerequisite Id="Microsoft.VisualStudio.InteractiveWindow" Version="[4.0.0.0,5.0.0.0)" DisplayName="Interactive Window" />
+    <Prerequisite Id="Microsoft.VisualStudio.Cache.Service" Version="[0.0,18.0)" DisplayName="Visual Studio Cache Service" />
```

Comment:
Should this just say min version 17, since that's what everything else does? ("When in Rome...")

Reply:
@AArnott ?

Reply:
Yes. 17.0 would be fine. It doesn't matter AFAIK as you'll always get the version for that VS version.